sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/logicaltypes/FixedPrecisionNumeric.java

```diff
@@ -113,4 +115,13 @@ public BigDecimal toInputType(BigDecimal base) {
     }
     return base;
   }
+
+  /**
+   * @TODO is byte or fixed?
+   * @TODO should is pass namespace/field name, etc?
+   * */
+  public org.apache.avro.Schema toAvroType() {
```

Comment:
We are in the process of getting rid of avro dependency in java core (CC: @aromanenko-dev) this function probably should goes to AvroUtils

Reply:
Fair. It needs to access `precision, scale)` which are private, yet, with no getters.